tests/test_convert_delta_to_json.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+# copyright 2022 mosaicml llm foundry authors
+# spdx-license-identifier: apache-2.0
+
+import os
+import sys
+from typing import Any
+
+# Add repo root to path so we can import scripts and test it
+repo_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.append(repo_dir)
+
+import unittest
+from unittest.mock import MagicMock, patch
+
+from scripts.data_prep.convert_delta_to_json import stream_delta_to_json
+
+
+class TestStreamDeltaToJson(unittest.TestCase):
```

Comment:
you shouldn't need to inherit from TestCase

```suggestion
class TestStreamDeltaToJson():
```

Reply:
Why?

Reply:
We don't do that for any of our other tests